doc/JSON_INFO.md

```diff
@@ -249,21 +249,38 @@ The syntax listed here is still valid.
 | `acid_resist`    | Ability of a material to resist acid.
 | `elec_resist`    | Ability of a material to resist electricity.
 | `fire_resist`    | Ability of a material to resist fire.
+| `dmg_adj`        | Adjectives used to describe damage states of a material.
 | `density`        | Density of a material.
+| `vitamins`       | Vitamins in a material. Usually overridden by item specific values.
+| `specific_heat_liquid` | Specific heat of a material when not frozen (J/(gK)). Default 4.186.
+| `specific_heat_solid`  | Specific heat of a material when frozen (J/(gK)). Default 2.108.
```

Comment:
Replace ``(gK)`` with ``(g K)``. Because ``g`` looks like a prefix now, something like gigaKelvin, that is possibly smaller then GigaKelvin (10^9 K).

Reply:
Done.